src/pages/design/[sectionId]/[articleId].tsx

```diff
@@ -27,17 +31,96 @@ export const getStaticProps: GetStaticProps = async (context) => {
 };
 
 export const ArticlePage = ({sectionId, articleId}: {sectionId: string; articleId: string}) => {
-    const section = sections.find((item) => item.id === sectionId);
+    const router = useRouter();
+    const section = designSections.find((item) => item.id === sectionId);
     const article = section?.articles.find((item) => item.id === articleId);
 
     if (!section || !article) {
         return null;
     }
 
+    const sections = useMemo<Section[]>(() => {
+        const result: Section[] = designSections.map((section) => ({
+            id: section.id,
+            title: section.title,
+            // Uncomment it to show overview tab
+            // url: `/design/${section.id}`,
+            subSections: section.articles.map((article) => ({
+                id: article.id,
+                title: article.title,
+                url: `/design/${section.id}/${article.id}`,
+            })),
+        }));
+        return result;
+    }, []);
+
+    const currentSection = useMemo(
+        () => sections.find((item) => item.id === sectionId),
+        [sectionId, sections],
+    );
+
+    const currentIndex = useMemo(() => {
+        if (!currentSection || !currentSection.subSections) {
+            return null;
+        }
+        return currentSection.subSections.findIndex((item) => item.id === articleId);
+    }, [currentSection, articleId]);
+
+    const nextSection = useMemo(() => {
```

Comment:
On the first page of design, the buttons look strange – http://localhost:3000/design/branding/resources

<img width="923" alt="image" src="https://github.com/gravity-ui/landing/assets/29424746/938ec00b-5aab-48b3-91a6-302729b8a11c">